### ISSUE TITLE
NEW FEATURE: project explorer

### ISSUE THREAD

**Issue by MuellerSeb** [psyplot/psy-view#12]
#### Summary
A project explorer like in spyder next to the psy-view app.

#### Reason
It would be nice to have a project explorer, that shows a certain directory-tree next to psy-view, to easily switch between files.

#### Examples
![img](https://docs.spyder-ide.org/_images/project_explorer_standard.png)

**Comment by Chilipp** [comment]
True, good idea! If you don't mind, I transfer this issue to psyplot-gui. It makes sense to implement this as a plugin for the general GUI and it can then be easily implemented for psy-view as well, as psyplot-gui is a dependency anyway.
